dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -0,0 +1,89 @@
+[[expanding_persistent_volumes]]
+= Expanding Persistent Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[enabling_expansion_pvc]]
+== Enabling Expansion of Persistent Volume Claims
+
+Volume expansion is a Technology Preview feature, and hence, is not enabled by
+default in your {product-title} {product-version} cluster. There may be other reasons that
+{product-title} administrators wish to enable this feature for certain use
+cases.
+
+[NOTE]
+====
+For more information on Red Hat Technology Preview features support scope,  see
+https://access.redhat.com/support/offerings/techpreview/. 
+====
+
+To allow expansion of persistent volume claims (PVC) by {product-title} users,
+{product-title} administrators must create or update a StorageClass with
+`allowVolumeExpansion` set to `true`. Only PVCs created from that class are
+allowed to expand.
+
+Apart from that, {product-title} administrators must enable the
+`ExpandPersistentVolumes` feature flag and turn on the
+`PersistentVolumeClaimResize` admission controller. Refer to <link_here>
+for how to enable a feature flag and how to enable an admission controller.
+
+[[expanding_glusterfs_pvc]]
+== Expanding GlusterFS-Based Persistent Volume Claims
+
+Expanding GlusterFS volumes is easiest. Once the  {product-title} administrator
+has created a StorageClass with `allowVolumeExpansion` set to `true`, you can
+create a PVC from that class, and afterwards, whenever needed, you can edit the
+PVC and request a new size. 
+
+For example:
+
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: gluster-mysql
+spec: <1>
+  storageClass: "storageClassWithFlagSet" 
+  accessModes:
+    - ReadWriteOnce
+  resources: <1>
+    requests: <1> 
+      storage: 8Gi
+----
+<1> You can request an expanded volume by updating `spec.resources.requests`.
+
```

Comment:
We should have `<1>` just next to `storage: 8Gi` line, because that is the line user will be actually editing.